ods_ci/tests/Tests/500__jupyterhub/custom-image.robot

```diff
@@ -97,7 +97,6 @@ Test Image From Local registry
     ...       ProductBug - https://github.com/opendatahub-io/odh-dashboard/issues/2185
```

Comment:
Same

Reply:
Interesting on this one is that it hasn't been fixed yet the test started to work? :thinking: